src/coverage.ts

```diff
@@ -360,7 +360,7 @@ export class Coverage {
           break;
 
         case 'IndexAccess':
-          kids = [ast.baseExpression, ast.indexExpression];
+          kids = [ast.baseExpression, ast.indexExpression].filter(_ => _);
```

Comment:
ah interesting nice find 💪 what's the syntax that maps to that?

Reply:
It was from this line in BaseBridgeReceiver:

```
(targets, values, signatures, calldatas) = abi.decode(
    data,
    (address[], uint256[], string[], bytes[])
);
```

`address[]` is an `IndexAccess` with no index expression